util/util.go

```diff
@@ -0,0 +1,291 @@
+package util
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"os/user"
+	"path/filepath"
+
+	"github.com/lbryio/lbry.go/extras/errors"
+	"github.com/lbryio/lbry.go/lbrycrd"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/filters"
+	"github.com/docker/docker/client"
+	"github.com/mitchellh/go-ps"
+	log "github.com/sirupsen/logrus"
+)
+
+func GetBlobsDir() string {
+	blobsDir := os.Getenv("BLOBS_DIRECTORY")
+	if blobsDir == "" {
+		usr, err := user.Current()
+		if err != nil {
+			log.Error(err.Error())
+			return ""
+		}
+		blobsDir = usr.HomeDir + "/.lbrynet/blobfiles/"
+	}
+
+	return blobsDir
+}
+
+func IsBlobReflectionOff() bool {
+	return os.Getenv("REFLECT_BLOBS") == "false"
+}
+
+func GetLBRYNetDir() string {
+	lbrynetDir := os.Getenv("LBRYNET_DIR")
+	if lbrynetDir == "" {
+		usr, err := user.Current()
+		if err != nil {
+			log.Errorln(err.Error())
+			return ""
+		}
+		return usr.HomeDir + "/.lbrynet/"
+	}
+	return lbrynetDir
+}
+
+const ALL = true
```

Comment:
all what?

Reply:
They are used for managing the containers. Its used in the parameter for docker api. you can get all containers or just those online. I figured `ALL` and `ONLINE` were better than `true`/`false`. I can add the word container though.